libsolidity/analysis/TypeChecker.cpp

```diff
@@ -1831,6 +1831,20 @@ bool TypeChecker::visit(MemberAccess const& _memberAccess)
 
 	if (exprType->category() == Type::Category::Contract)
 	{
+		// Warn about using address members on contracts
+		bool v050 = m_scope->sourceUnit().annotation().experimentalFeatures.count(ExperimentalFeature::V050);
```

Comment:
`const`? :)

Reply:
Oh come on!

Reply:
I don't care about this, you were putting that on other reviews :)

Reply:
For such cases?

Reply:
Yeah.